### PR TITLE
HttpAdapter docs

### PR DESCRIPTION
**Pull request by HerrMuellerluedenscheid**
Sane version of #137 addressing issue #129

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ Documentation
 =============
 PyOTA's documentation is available on `ReadTheDocs`_.
 
-If you are :ref:`installing from source <readme-installing-from-source>`, you
+If you are [installing from source](#installing-from-source), you
```
Comment (todofixthis) [inline README.rst]: The previous syntax is actually correct for `.rst` files.  I'll file a request with GitHub support to figure out why it's not rendering correctly.

File: docs/adapters.rst [hunk 2]
```diff
@@ -44,6 +44,14 @@ HttpAdapter
     api = Iota('https://service.iotasupport.com:14265')
     api = Iota(HttpAdapter('https://service.iotasupport.com:14265'))
 
+    # Use HTTPS with basic authentication and 60 seconds timeout:
+    api = Iota('https://USERNAME:MYIRIPASSWORD@service.iotasupport.com:14265',
+               timeout=60)
```
Comment (todofixthis) [inline docs/adapters.rst]: ❔ I'm not sure that this works — I believe the `timeout` argument only works with an `HttpAdapter` instance.

File: docs/adapters.rst [hunk 3]
```diff
@@ -44,6 +44,14 @@ HttpAdapter
...
+    api = Iota(
+        HttpAdapter(
+            'https://USERNAME:MYIRIPASSWORD@service.iotasupport.com:14265',
+            timeout=60))
```
Comment (todofixthis) [inline docs/adapters.rst]: 👍 Looks great!

File: docs/adapters.rst (hunk 2 above)
Comment (HerrMuellerluedenscheid) [reply docs/adapters.rst]: Yes, that was a bit sloppy, copy-paste...

File: README.rst (hunk 1 above)
Comment (todofixthis) [reply README.rst]: Hey Marius.  I heard back from GitHub support.

Turns out that `:ref:` is not actually part of the reStructuredText spec; it's a custom feature provided by Sphinx.

I made the following changes:
- Removed the link.
- Added a metadata check to `.travis.yml` (which caught the `:ref:` right awy!).

Thanks for drawing attention to this issue!

**Pull request merged**